src/StackExchange.Redis/RedisKey.cs

```diff
@@ -13,7 +13,7 @@ namespace StackExchange.Redis
     {
         internal RedisKey(byte[]? keyPrefix, object? keyValue)
         {
-            KeyPrefix = keyPrefix?.Length == 0 ? null : keyPrefix;
```

Comment:
Let's compare using:

``` c#
using System;

Original(null);
Original("");
Original("abc");

Proposed(null);
Proposed("");
Proposed("abc");

static void Original(string? keyPrefix) => Write(keyPrefix?.Length == 0 ? null : keyPrefix);
static void Proposed(string? keyPrefix) => Write(keyPrefix == null || keyPrefix.Length == 0 ? null : keyPrefix);

static void Write(string? value)
    => Console.WriteLine(value is null ? "(null)" : "\"" + value + "\"");
```

outputs:

keyPrefix input | original output | proposed output
---|---|---
null | null | null
"" | null | null
"abc" | "abc" | "abc"


So... this has no functional change